### DISCUSSION TITLE
would be cool if you could include A-ESRGAN:

### DISCUSSION THREAD

**Discussion by Bahaduraly** [rkhamilton/vqgan-clip-generator#69] [Ideas]
Training Real-World Blind Super-Resolution with Attention-based U-net Discriminators
https://github.com/aesrgan/A-ESRGAN/blob/main/README.md

**Comment by rkhamilton** [comment]
I’m not interested in creating a Swiss Army knife toolkit combining many unrelated tools. My library is intended to be part of a python image generation tool chain, not a single solution (i.e., it is not a Jupyter notebook). You should be able to follow the examples that I’ve provided with Real-ESRGAN to integrate any up-scaling / image restoration technology you like in your own tool chain.
